### DISCUSSION TITLE
No module named 'colorlog'

### DISCUSSION THREAD

**Discussion by ruslaniv** [borntyping/python-colorlog#117] [Support] [answered]
I have installed colorlog as:
`pipenv install colorlog --dev` 
Pipfile lists the following:
```
[dev-packages]
flake8 = "*"
mypy = "*"
colorlog = "*"
```
Then I import the package in the .py script
`import colorlog`
But when I run the script I get the error 
`ModuleNotFoundError: No module named 'colorlog'`

I'm running python 3.8.1 on macOS 10.15.7

How can I troubleshoot the reason for this problem?

**Comment by borntyping** [comment, marked as answer]
You are likely running your code with a different Python executable, or not using the virtualenv Pipenv manages (e.g. by using `pipenv run script.py`).

(Closing this as it's not a colorlog issue.)
